sdk/attestation/Microsoft.Azure.Management.Attestation/AzSdk.RP.props

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
   <PropertyGroup>
-    <AzureApiTag />
+    <AzureApiTag>Attestation_2018-09-01-preview;</AzureApiTag>
```

Comment:
Any particular reason to include the change in this Microsoft.ResourceManager related PR?

Reply:
Not sure how did they changes get here. Will revert.